google/cloud/aiplatform/constants.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+DEFAULT_REGION = "us-central1"
+SUPPORTED_REGIONS = ("us-central1", "europe-west4", "asia-east1")
+PROD_API_ENDPOINT = "aiplatform.googleapis.com"
```

Comment:
but this is not really an api endpoint.

Reply:
Does `PROD_API_BASE_PATH` or `PROD_API_BASE_URL` work?